components/aws/sagemaker/tests/integration_tests/conftest.py

```diff
@@ -136,7 +137,11 @@ def boto3_session(region):
 
 @pytest.fixture(scope="session")
 def sagemaker_client(boto3_session):
-    return boto3_session.client(service_name="sagemaker")
+    return boto3_session.client(
+        service_name="sagemaker",
+        # Increase max_attempts to avoid ThrottlingException
```

Comment:
were you experiencing a significant amount of  throttlingExceptions at boto3 client creation? That seems unusual..

Reply:
can we revert this change? if we are hitting this, we should consider removing the calls that are not needed, redesigning out test suite or decrease test parallelism

Reply:
We are having the throttling error because we have 4 job definition test running in parallel. Each of them will trigger CreateModel. If we delete this, one of the job definition will fail because of the ThrottlingException